src/routes/api/discord/[userId].ts

```diff
@@ -0,0 +1,30 @@
+import { Routes } from 'discord-api-types/v10'
+import type { APIGuildMember } from 'discord-api-types/v10'
+import { api } from '$lib/discord'
+import type { RequestHandler } from '@sveltejs/kit'
+
+const guildId = import.meta.env.VITE_DISCORD_GUILD_ID
+
+async function getDiscordUsername(userId: string) {
+  try {
+    const guildMember = (await api.get(Routes.guildMember(guildId, userId))) as
+      | APIGuildMember
+      | undefined
+    if (guildMember?.nick) return guildMember.nick
+    if (guildMember?.user?.username) return guildMember.user.username
+  } catch (error) {
+    // console.error(`Failed to fetch discord username: ${error.message}`)
```

Comment:
should this be uncommented or throw?

Reply:
Yeah should be uncommented..it was just getting annoying because the none of the seeded users have a username